updating/updating-cluster-between-minor.adoc

```diff
@@ -0,0 +1,24 @@
+[id="updating-cluster-between-minor"]
+= Updating a cluster between minor versions
+include::modules/common-attributes.adoc[]
+:context: updating-cluster-between-minor
+
+toc::[]
+
+You can update, or upgrade, an {product-title} cluster between minor versions.
+
+[NOTE]
+====
+Because of the difficulty of changing update channels by using `oc`, you use the web console to change the update channel. It is recommended to complete the update process within the web console. You can follow the steps in xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] to complete the update after you change to a {product-version} channel.
```

Comment:
"you use the web console" sounds weird to me. I would drop "you" and just say "use the web console"

Reply:
I think there was a "must" in there at one point. Thanks!